### PR TITLE
Sync Overview Tree view prod code with pattern library

### PR DESCRIPTION
**Pull request by line47**
Updated the Space Tree View with code from prod into the PL. 
Also updated the code for the card component from prod code.

### CODE REVIEW

File: docs/src/components/components/status/status.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-<span class="status status-{{ modifier }}">{{ text }}</span>{{ patternbreak }}
```
Comment (line47) [inline docs/src/components/components/status/status.html:2]: Removed this as it doesn't work when including other components.

File: docs/src/components/features/Tree View/space-tree-view.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<div class="panel-row">
```
Comment (line47) [inline docs/src/components/features/Tree View/space-tree-view.html:1]: Updated this to reflect the code that is currently in production.

File: docs/src/components/features/card/card.config.yml [hunk 3]
```diff
@@ -1,29 +1,10 @@
...
-      isOpen: true
-      spaces:
-        - space:
-          name: 'bar/foo'
     - org:
       name: 'sandbox'
-      isOpen: false
-      spaces:
-        - space:
-          name: 'sandbox/user.name'
```
Comment (line47) [inline docs/src/components/features/card/card.config.yml:29]: simplified the card panel and removed variables that were not being used.

File: docs/src/components/features/card/card.html [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
     </div>
-
-    {% if isOpen %}
-      <div class="card-items">
-        {% render '@overview-row' %}
-      </div>
-    {% endif %}
-  </div>
+</div>
 {% endfor %}
```
Comment (line47) [inline docs/src/components/features/card/card.html:19]: Updating this pattern to match prod code.

File: docs/src/components/features/overview/overview.config.yml [hunk 5]
```diff
@@ -1,2 +0,0 @@
-title: overview (contents)
-status: wip
```
Comment (line47) [inline docs/src/components/features/overview/overview.config.yml:2]: Removed overview and changed to `space-tree-view`

File: fractal.js [hunk 6]
```diff
@@ -23,8 +23,8 @@ fractal.components.set('statuses', {
     description: "Code implementation is in exploratory phase, incomplete and not for use.",
     color: "#be4900"
   },
-  wip: {
-    label: "WIP",
+  draft: {
+    label: "Draft",
```
Comment (line47) [inline fractal.js:27]: Changed `WIP` to `Draft` at the root level and also updated many components.

**Review by thisisdano — APPROVED** [review]

**Pull request merged**